v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/transforms/Neo4jRowWriterTransform.java

```diff
@@ -91,6 +91,31 @@ public PCollection<Row> expand(PCollection<Row> input) {
         .setRowSchema(input.getSchema());
   }
 
+  private void createIndicesAndConstraints() {
+    List<String> cyphers = generateIndexAndConstraints();
+    if (cyphers.isEmpty()) {
+      return;
+    }
+    Neo4jConnection neo4jDirectConnect = new Neo4jConnection(neoConnection);
+    LOG.info("Adding {} indices and constraints", cyphers.size());
+    for (String cypher : cyphers) {
+      LOG.info("Executing cypher: {}", cypher);
+      try {
+        neo4jDirectConnect.executeCypher(cypher);
+      } catch (Exception e) {
+        LOG.error("Error executing cypher: {}, {}", cypher, e.getMessage());
```

Comment:
nit This would log the entire stacktrace

```suggestion
        LOG.error("Error executing cypher: {}", cypher, e);
```